.github/workflows/build-electron-production.yml

```diff
@@ -14,10 +14,9 @@ jobs:
       CONFIG_FILE: config.json
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
-        with:
-          node-version: '16'
+        uses: actions/checkout@v3
+      - name: Set up node with volta
+        uses: volta-cli/action@v4
```

Comment:
Shouldn't we add the "volta pin" commit then into this MR?

https://github.com/neopostmodern/wohnungsbot/pull/156/commits/167ac633418652892ae3045468779f4e3b5c70ff

That's why you did the shift from setup-node to volta, right?

Reply:
Good catch, I cherry-picked that.